cns/cnsclient/cnsclient_test.go

```diff
@@ -30,18 +30,26 @@ var (
 	}
 )
 
-// func addTestStateToRestServer(svc *restserver.HTTPRestService) {
-// 	// set state as already allocated
-// 	state1, _ := restserver.NewPodStateWithOrchestratorContext(testIP1, 24, testPod1GUID, testNCID, cns.Available, testPod1Info)
-// 	ipconfigs := map[string]cns.SecondaryIPConfig{
-// 		state1.ID: state1,
-// 	}
-// 	nc := cns.CreateNetworkContainerRequest{
-// 		SecondaryIPConfigs: ipconfigs,
-// 	}
-
-// 	svc.CreateOrUpdateNetworkContainerWithSecondaryIPConfigs(nc)
-// }
+func addTestStateToRestServer(svc *restserver.HTTPRestService) {
+	state1 := cns.SecondaryIPConfig{
+		IPConfig: cns.IPSubnet{
+			IPAddress:    testIP1,
+			PrefixLength: 24,
+		},
+	}
+	ipconfigs := map[string]cns.SecondaryIPConfig{
+		testPod1GUID: state1,
+	}
+	nc := cns.CreateNetworkContainerRequest{
+		SecondaryIPConfigs: ipconfigs,
+	}
+
+	err := svc.CreateOrUpdateNetworkContainerWithSecondaryIPConfigs(nc)
```

Comment:
This api is removed

Reply:
Can you comment this test for now, we will add it when my pr gets in

Reply:
fixed